tooling/cli/templates/app/src-tauri/Cargo.crate-manifest

```diff
@@ -20,5 +20,9 @@ serde = { version = "1.0", features = ["derive"] }
 tauri = {{{  tauri_dep  }}}
 
 [features]
+# by default Tauri runs on production mode
```

Comment:
Although (I just learned) both "in production mode" and "on production mode" would be fine (I think) "in" sounds way nicer to me 🤔 
example "you have to run this application in compatibility mode"

Reply:
feels like that's because we're german lol

Reply:
riiiight, but I actually googled this 😂 It's not just my brain liking it more, the internet is on my side here 😂